packages/store/src/public_api.ts

```diff
@@ -25,7 +25,9 @@ export {
   StateOperator,
   NgxsOnInit,
   NgxsAfterBootstrap,
-  NgxsModuleOptions
+  NgxsOnChanges,
+  NgxsModuleOptions,
+  NgxsSimpleChange
```

Comment:
Add trailing comma. Makes futher changes easier to merge.

Reply:
Prettier remove before pushed

Reply:
Meh